webpack.config.js

```diff
@@ -44,6 +37,17 @@ const webpackConfig = {
   }
 };
 
+if (isDevelopment()) {
+  // In local development, use the debugger as a single bundle
+  webpackConfig.entry.debugger = getEntry("src/main.js");
+} else {
```

Comment:
@jasonLaster When we are not in development, we no longer build the debugger bundle, but we still build other bundles. Maybe we can simplify some things here?